cookbooks/composition-reaction/doc/composition-reaction.md

```diff
@@ -1,4 +1,4 @@
-## The active case with reactions.
+# The active case with reactions.
```

Comment:
Why do we need to make this change? Logically they are subchapters to  https://aspect-documentation.readthedocs.io/en/latest/user/cookbooks/cookbooks-composition.html or not?

Reply:
They are, but Sphinx expects all documents to start with a `#` title. This means the `#` levels determine the rendering and logical ordering of headings on a particular page (or one markdown file), and the appearance of the file within the `toctree` sections determines where it is logically in the global table of contents.